packages/components/src/components/banner/banner.tsx

```diff
@@ -67,24 +67,26 @@ export class Banner {
 
   public render(): JSX.Element {
     validateProps(this, propTypes);
-    attachComponentCss(this.host, getComponentCss);
+    const deprecatedStateMap: Partial<Record<InlineNotificationState, InlineNotificationState>> = {
+      neutral: 'info',
+    };
+    warnIfDeprecatedPropValueIsUsed(this.host, 'state', deprecatedStateMap);
+    attachComponentCss(this.host, getComponentCss, this.width);
 
     const PrefixedTagNames = getPrefixedTagNames(this.host);
 
     return (
-      <PrefixedTagNames.pContentWrapper width={this.width}>
-        <PrefixedTagNames.pInlineNotification
-          ref={(el) => (this.inlineNotificationElement = el)}
-          class="root"
-          state={this.state}
-          persistent={this.persistent}
-          theme={this.theme}
-          onDismiss={this.removeBanner}
-        >
-          {hasNamedSlot(this.host, 'title') && <slot name="title" slot="heading" />}
-          {hasNamedSlot(this.host, 'description') && <slot name="description" />}
-        </PrefixedTagNames.pInlineNotification>
-      </PrefixedTagNames.pContentWrapper>
+      <PrefixedTagNames.pInlineNotification
+        ref={(el) => (this.inlineNotificationElement = el)}
+        class="root"
+        state={this.state}
+        persistent={this.persistent}
+        theme={this.theme}
+        onDismiss={this.removeBanner}
+      >
+        {hasNamedSlot(this.host, 'title') && <slot name="title" slot="heading" />}
```

Comment:
`title` and `description` props are being added later?

Reply:
added to v3 status docs